### PR TITLE
docs: fix broken relative links in docs

### PR DESCRIPTION
**Pull request by AlvaroBalbin**
I fixed a handful of markdown links that pointed at files in the wrong place. The workload generator and client READMEs were linking to each other one directory level short, the standalone deployment README's LICENSE link was missing a level too, the vtc_readme link to the e2e test was off by one directory, the melange solver.py link pointed at a nonexistent nested path, and the deepseek-r1 dashboard link broke because an unescaped parenthesis in the filename was cutting the URL short.

The checks evaluated by the fork CI gate passed. This excludes skipped checks and checks filtered out for fork-only conditions.

### CODE REVIEW

**Comment by varungup90** [comment]
@AlvaroBalbin please sign the PR

**Comment by AlvaroBalbin** [comment]
Thanks for the reminder. I have left the documentation fix unchanged. Signing requires the contributor’s own action, which I cannot perform on their behalf. This follow-up was AI-assisted. I did not run build, lint, or tests because I made no changes.

File: pkg/plugins/gateway/algorithms/vtc_readme.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ Incoming Request (with user identity)
 └──────────────────────────────────┘
 ```
 
-End-to-end tests: [vtc_routing_test.go](../../../test/e2e/vtc_routing_test.go)
+End-to-end tests: [vtc_routing_test.go](../../../../test/e2e/vtc_routing_test.go)
```
Comment (varungup90) [inline pkg/plugins/gateway/algorithms/vtc_readme.md:79]: This link is still broken. `vtc_routing_test.go` doesn't live at `test/e2e/vtc_routing_test.go` — it's at `test/e2e/gateway/routing/vtc_routing_test.go`. From `pkg/plugins/gateway/algorithms/`, that means going up 4 directories to the repo root and then into `test/e2e/gateway/routing/`:

```
../../../../test/e2e/gateway/routing/vtc_routing_test.go
```

(4 `../` gets you from `algorithms/` to the repo root; the current fix's 4 `../` lands at the root correctly, but the path after it is missing the `gateway/routing/` segment.)

---
_Generated by [Claude Code](https://claude.ai/code)_

**Review by varungup90 — CHANGES_REQUESTED** [review]
Reviewed the 6 link fixes against the actual repo layout. 5 of 6 are correct (workload generator README, client README, standalone LICENSE link, melange solver.py, and the deepseek-r1 dashboard URL-encoding fix all resolve to real files). One link is still broken — left an inline comment with the fix.

Also noting the DCO check is still failing (missing sign-off on commit `4d4bb1b`) — that's a separate item already being tracked in the PR thread.

---
_Generated by [Claude Code](https://claude.ai/code)_

**Comment by varungup90** [comment]
> Thanks for the reminder. I have left the documentation fix unchanged. Signing requires the contributor’s own action, which I cannot perform on their behalf. This follow-up was AI-assisted. I did not run build, lint, or tests because I made no changes.
> 

You can amend current commit with sign and force push.
